CHANGELOG.asciidoc

```diff
@@ -3,6 +3,125 @@
+- Fix duplicated addition of regexp extension in CEL input. {pull}38181[38181]
+- Fix HTTPJSON handling of empty object bodies in POST requests. {issue}33961[33961] {pull}38290[38290]
+- Fix PEM key validation for CEL and HTTPJSON inputs. {pull}38405[38405]
+
+*Heartbeat*
+
+- Adjust State loader to only retry when response code status is 5xx. {pull}37981[37981]
+
+*Metricbeat*
+
+- Fix Azure Monitor 429 error by causing metricbeat to retry the request again. {pull}38294[38294]
+- Fix fields not being parsed correctly in postgresql/database. {issue}25301[25301] {pull}37720[37720]
+
+==== Added
+
+*Affecting all Beats*
+
+- Ignore Kubernetes node and namespace update events that do not change pod metadata. {issue}37338[37338] {pull}37431[37431]
+- Enhance add_cloud_metadata processor with `orchestrator.cluster.name`, `orchestrator.cluster.id` and `azure.resourcegroup.name` when running inside an AKS cluster. {issue}33081[33081] {pull}37685[37685]
+- Upgrade go-sysinfo from 1.12.0 to 1.13.1. {pull}37996[37996]
+- Make `range` condition work with numeric values as strings. {pull}38080[38080]
+- Allow users to configure number of output workers (for outputs that support workers) with either `worker` or `workers`. {pull}38257[38257]
+- Kafka output now validates the `topics` and `topic` configuration values. {pull}38058[38058]
+
+*Auditbeat*
+
+- Add linux capabilities to processes in the system/process. {pull}37453[37453]
+- Add opt-in eBPF backend for file_integrity module. {pull}37223[37223]
+
+*Filebeat*
+
+- Update SQL input documentation regarding Oracle DSNs {pull}37590[37590]
+- Add support for complete URL replacement in HTTPJSON chain steps. {pull}37486[37486]
+- Add support for user-defined query selection in EntraID entity analytics provider. {pull}37653[37653]
+- Update CEL extensions library to v1.8.0 to provide runtime error location reporting. {issue}37304[37304] {pull}37718[37718]
+- Add request trace logging for chained API requests. {issue}37551[36551] {pull}37682[37682]
+- Add support for PEM-based Okta auth in HTTPJSON. {pull}37772[37772]
+- Prevent complete loss of long request trace data. {issue}37826[37826] {pull}37836[37836]
+- Added experimental version of the Websocket Input. {pull}37774[37774]
+- Add support for PEM-based Okta auth in CEL. {pull}37813[37813]
+- Add ETW input. {pull}36915[36915]
+- Update CEL mito extensions to v1.9.0 to add keys/values helper. {pull}37971[37971]
+- Add parseDateInTZ value template for the HTTPJSON input. {pull}37738[37738]
+- Improve rate limit handling by HTTPJSON. {issue}36207[36207] {pull}38161[38161] {pull}38237[38237]
+
+*Libbeat*
+- Add watcher that can be used to monitor Linux kernel events. {pull}37833[37833]
+- Added support for ETW reader. {pull}36914[36914]
+
+*Heartbeat*
+- Upgrade github.com/elastic/go-elasticsearch/v8 to v8.12.0. {pull}37673[37673]
+
+*Metricbeat*
+
+- Fix containerd metrics grouping for TSDB {pull}37537[37537]
```

Comment:
```suggestion
- Fix containerd metrics grouping for TSDB. {pull}37537[37537]
```
Just for consistency.

Reply:
I thought I took care of all missing dot, good catch thx!!!